src/components/AppView/AppNotes.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+
+interface IAppNotesProps {
+  notes?: string | null;
+}
+
+class AppNotes extends React.PureComponent<IAppNotesProps> {
```

Comment:
Cool! Glad to see `PureComponents` on the code :)

Reply:
;) I guess by this comment it makes sense to use that here? I'm still not that sure when to use it.

Reply:
Yes! We can have a small talk about it :)